jgrapht-core/src/test/java/org/jgrapht/graph/SimpleIdentityDirectedGraphTest.java

```diff
@@ -97,8 +97,8 @@ public SimpleIdentityDirectedGraph(EdgeFactory<V, E> ef) {
         }
 
         @Override
-        protected DirectedSpecifics createDirectedSpecifics() {
-            return new DirectedSpecifics(new IdentityHashMap<V, AbstractBaseGraph.DirectedEdgeContainer<V, E>>());
+        protected DirectedSpecifics<V,E> createDirectedSpecifics() {
```

Comment:
@cwensel:  the refactoring Joris has done here would be a breaking change for you.  Normally we try to avoid this via deprecated etc...is it important in this case?